src/headingengine.js

```diff
@@ -30,6 +23,39 @@ export default class HeadingEngine extends Plugin {
 	/**
 	 * @inheritDoc
 	 */
+	constructor( editor ) {
+		super( editor );
+
+		const t = editor.t;
+
+		/**
+		 * A set of default localized labels for `config.heading.formats`.
+		 *
+		 * @readonly
+		 * @protected
+		 * @member {Object} #_localizedFormatLabels
+		 */
+		const labels = this._localizedFormatLabels = {
```

Comment:
It's an edge case, really. When someone translates the entire app but some labels.